python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -85,6 +85,25 @@ def MmDagModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(4, 4), tu.rand(4, 4))
 
 
+# ==============================================================================
+
+class ContainsIntList(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+    ])
+    def forward(self,):
+        return 3 in [1,2,3]
+
+
+@register_test_case(module_factory=lambda: ContainsIntList())
+def ContainsIntList_basic(module, tu: TestUtils):
+    module.forward()
+
+
```

Comment:
Did you try using the `torch.ops.aten.__contains__` for this test? Also, you should add a test for the case which evaluates to `false`.

Reply:
Maksim explained to me that the torch.ops.aten__contains function effectively replaces "return 3 in [1,2,3]". If it doesn't, I can replace the code.

Reply:
That's probably the case, but for these tests we prefer the `torch.ops.aten...` version because it makes it very clear what is being tested